PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -327,6 +335,226 @@ struct HfCandidateCreatorToXiPi {
 
     } // loop over LF Cascade-bachelor candidates
   }   // end of process
+  PROCESS_SWITCH(HfCandidateCreatorToXiPi, processWithCascades, "Use regular cascade information", true);
+
+  void processWithTrackedCascades(aod::Collisions const&,
+                                  aod::BCsWithTimestamps const&,
+                                  TracksWCovDca const&,
+                                  MyTrkCascTable const&, CascadesLinked const&,
+                                  MySkimIdx const& candidates)
+  {
+    double massPionFromPDG = MassPiPlus;    // pdg code 211
+    double massLambdaFromPDG = MassLambda0; // pdg code 3122
+    double massXiFromPDG = MassXiMinus;     // pdg code 3312
+    double massOmegacFromPDG = MassOmegaC0; // pdg code 4332
+    double massXicFromPDG = MassXiC0;       // pdg code 4132
```

Comment:
Why is this needed? Can't you directly use the variables on the right of the ` = ` sign?